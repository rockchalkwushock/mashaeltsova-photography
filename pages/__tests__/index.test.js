import React from 'react'
import { shallow, render } from 'enzyme'

import Index from '../'

describe('Index Page', () => {
  test('shallow', () => {
    const wrapper = shallow(<Index />)
    expect(wrapper.text()).toEqual('Hello World!')
  })

  test('render', () => {
    const wrapper = render(<Index />)
    expect(wrapper.text()).toEqual('Hello World!')
  })

  test('snapshot', () => {
    const tree = render(<Index />)
    expect(tree).toMatchSnapshot()
  })
})
