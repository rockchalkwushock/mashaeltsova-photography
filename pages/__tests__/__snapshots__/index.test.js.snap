// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page snapshot 1`] = `
<h1>
  Hello World!
</h1>
`;
